sample_apps/run_example.sh

```diff
@@ -2095,6 +2174,40 @@ function setup_app_by_name() {
     run_steps "build_simple_server" "provision_app_service_files"
 }
 
+# ###########################################################################
+# This is very similar to the steps followed for most other apps, as
+# implemented by setup_app_by_name(). -BUT- as we are still dealing with a
+# Keystone-shim, 'simpleserver' is built with the emulated key/cert files.
+# So, we need to provision those first before building the Certifier service.
```

Comment:
This rearrangement that we provision `service/` dir before building `simpleserver` is due to the fact that we are hard-coding some emulated_keystone bin file's name in the Go source code.